lib/ews/soap/parsers/ews_parser.rb

```diff
@@ -28,22 +28,24 @@ def initialize(soap_resp)
 
     def parse(opts = {})
       opts[:response_class] ||= EwsSoapResponse
-      sax_parser.parse(sanitize_response(@soap_resp))
+      sax_parser.parse(clean_xml(@soap_resp))
       opts[:response_class].new @sax_doc.struct
     end
 
-    #
-    # We empirically encountered a &#x10; in the wild which is an invalid XML character.
-    # In order to parse correctly we need to remove
-    #
-    def sanitize_response(resp)
```

Comment:
Didn't realize this was in here until I rebased against our branch, it's a serious bummer that this project isn't actively maintained.